### PR TITLE
add dark-theme to PNI header + intro text

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/6185 by adding a `.dark-theme` rule to the PNI header and intro text, with a toggle in the CMS that lets you turn it on or off as needed.

live URL: https://foundation-s-pni-dark-m-pli3qf.herokuapp.com/en/privacynotincluded/
CMS url: https://foundation-s-pni-dark-m-pli3qf.herokuapp.com/cms/pages/103/edit/

### CODE REVIEW

File: source/sass/buyers-guide/views/catalog.scss [hunk 1]
```diff
@@ -35,6 +35,10 @@ body.catalog {
         padding-top: 90px;
         padding-bottom: 90px;
       }
+
+      .dark-theme {
+        color: $white;
+      }
```
Comment (Pomax) [inline source/sass/buyers-guide/views/catalog.scss:41]: I tried using this, but that didn't want to work:
```css
      h1 {
        @include text-base-style("Zilla Slab", 300, $black, $white);
      }

      p {
        @include text-base-style("Nunito Sans", 300, $black, $white);
      }
```

Comment (mmmavis) [reply source/sass/buyers-guide/views/catalog.scss:41]: We shouldn't need any additional SCSS here. Having `dark-theme` on `intro-text` should just get the right styling to kick in! 😄

Comment (mmmavis) [reply source/sass/buyers-guide/views/catalog.scss:41]:
```suggestion
```

**Review by natalieworth — DISMISSED** [review]
Design is great! Thank you!!!! 🙌

**Review by mmmavis — CHANGES_REQUESTED** [review]
No additional SCSS needed!

File: source/sass/buyers-guide/views/catalog.scss [hunk 2]
```diff
@@ -35,6 +35,7 @@ body.catalog {
         padding-top: 90px;
         padding-bottom: 90px;
       }
+
     }
```
Comment (Pomax) [inline source/sass/buyers-guide/views/catalog.scss:39]:
```suggestion
    }
```

**Review by mmmavis — APPROVED** [review]
woohoo!

**Pull request merged**